frontend/src/component/admin/LazyAdminExport.tsx

```diff
@@ -0,0 +1,7 @@
+// Lazy loading only supports default export. We have an ADR
```

Comment:
We can do it without default export: https://dev.to/iamandrewluca/react-lazy-without-default-export-4b65

Reply:
I tried this approach first, but couldn't get it to work. Let me have another look before merging.